docs/configuration.md

```diff
@@ -284,16 +298,22 @@ When running Lighthouse CI on a page with behind authentication, you'll need to
  * @param {{url: string, options: LHCI.CollectCommand.Options}} context
  */
 module.exports = async (browser, context) => {
+  // launch browser for LHCI
   const page = await browser.newPage();
   await page.goto('http://localhost:8080/login');
   await page.type('#username', 'admin');
   await page.type('#password', 'password');
   await page.click('[type="submit"]');
   await page.waitForNavigation();
+  // we're logged in, so test the page behind auth
+  // runs the url set in lighthouserc
+  await page.goto(context.url);
```

Comment:
this shouldn't be necessary because Lighthouse is going to open a new tab and navigate to this page immediately after this script finishes.

was there something you ran into that prompted this?

Reply:
yeh, I could be very wrong though. Somehow I got to a point where this was necessary because the page was empty and the LHCI run would simply time out. I threw this in there and it worked.

Regardless, I agree this probably isn't necessary as a generic example!